### DISCUSSION TITLE
Builder API to get a data object that can be used directly by Vueform?

### DISCUSSION THREAD

**Discussion by djkunkel** [vueform/builder#30] [Questions] [answered]
Is there a way to ask the Builder to give you a schema object (and any steps or tabs that might be defined) that would be suitable to use directly by Vueform in this way: https://vueform.com/docs/rendering-forms#using-schema-object

I'm able to save and load the Builder model so that users can continue editing a previously saved form, but I can't figure out how to get the builder to give me a JSON document/object I can use directly with Vueform without copy and pasting the object from the Export tab.

Background:
I have a component for our CMS that renders a JSON document that my custom Vueform using Vue component uses to render a form, and I'd like to be able to have the builder give me an appropriate document for later use.

Thank you!

**Comment by adamberecz** [comment, marked as answer]
The Builder provides the following object upon saving:

```js
let builderObject = {
  "schema": {
    // elements
  },
  "theme": {
    // theme variables
  },
  "form": {
    // form options (including steps, if any)
  },
  "export": {
    // builder export settings
  },
  "builder": {
    // builder settings
  }
}
```

You can reach the form's elements (`schema`) and form options (`form`) via it and pass them to a `<Vueform>` component:

```html
<Vueform v-bind="builderObject.form" :schema="builderObject.schema">
```

I put together a short chapter in our Docs explaining Rendering:
https://builder.vueform.com/docs/rendering
